### ISSUE TITLE
Tags for warmup lambda function

### ISSUE THREAD

**Issue by chaitanya11** [FidelLimited/serverless-plugin-warmup#49]
It would be great if we can add tags to warmup lambda function. 
like this :
  ```
warmup:
    cleanFolder: false
    memorySize: 256
    name: 'sample_warmup'
    schedule: 'rate(3 minutes)'
    timeout: 20
    prewarm: true
    folderName: '_warmup'
    tags: # Function specific tags
      Project: sampleproject
      Owner: foo bar
      Environment: dev
```

**Comment by chaitanya11** [comment]
@goncaloneves can we expect this functionality up coming version ?.

**Comment by juanjoDiaz** [comment]
This can be closed by #51

**Comment by goncaloneves** [comment]
Very true. Thanks. 👌
